src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -319,10 +318,7 @@ public static String sanitizeForCsv(String str) {
     public static List<String> sanitizeListForCsv(List<String> strList) {
         List<String> sanitizedStrList = new ArrayList<>();
 
-        Iterator<String> itr = strList.iterator();
-        while (itr.hasNext()) {
-            sanitizedStrList.add(sanitizeForCsv(itr.next()));
-        }
+        strList.forEach(string -> sanitizedStrList.add(sanitizeForCsv(string)));
```

Comment:
Perhaps you can remove the temporary variable entirely and use `map` to make this method a one-liner?

Reply:
You're right, we can remove the temporary data structure by mapping sanitizeForCsv to the strList. I have committed the changes.

OLD

```
List<String> sanitizedStrList = new ArrayList<>();
strList.forEach(string -> sanitizedStrList.add(sanitizeForCsv(string))); 
return sanitizedStrList;
```

NEW

```
 return strList.stream().map(string -> sanitizeForCsv(string)
                        .collect(Collectors.toList());
```